charts/atlantis/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v0.20.1
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.2.0
+version: 4.3.0
```

Comment:
You may hit a conflict here with some competing PRs.

Reply:
If https://github.com/runatlantis/helm-charts/pull/203 is good to go I'll bump to 4.4.0